cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -261,6 +268,14 @@ protected void setMetricsEnabled(boolean isMetricsEnabled) {
         this.isMetricsEnabled = isMetricsEnabled;
     }
 
+    public boolean isGcLoggingDisabled() {
+        return gcLoggingDisabled;
+    }
+
+    protected void setGcLoggingDisabled(boolean gcLoggingDisabled) {
+        this.gcLoggingDisabled = gcLoggingDisabled;
+    }
+
```

Comment:
Rather than having `public boolean isGcLoggingDisabled()` why not just have a method `public String getGcLoggingOptions()` which you could define here in `AbstractModel` and override in `KafkaCluster` to provide the Kafka-specific options. This would avoid all the futzing around with ternary operators in each of the `AbstractModel` subclasses.